fe/fe-core/src/main/java/org/apache/doris/analysis/ShowCatalogStmt.java

```diff
@@ -0,0 +1,93 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.analysis;
+
+import org.apache.doris.catalog.Column;
+import org.apache.doris.catalog.ScalarType;
+import org.apache.doris.common.AnalysisException;
+import org.apache.doris.common.Config;
+import org.apache.doris.common.UserException;
+import org.apache.doris.qe.ShowResultSetMetaData;
+
+/**
+ * Statement for show all catalog or desc the specific catalog.
+ */
+public class ShowCatalogStmt extends ShowStmt {
+    private static final ShowResultSetMetaData META_DATA_ALL =
+            ShowResultSetMetaData.builder()
+                    .addColumn(new Column("CatalogName", ScalarType.createVarchar(30)))
+                    .addColumn(new Column("Type", ScalarType.createVarchar(30)))
+                    .build();
+
+    private static final ShowResultSetMetaData META_DATA_SPECIFIC =
+            ShowResultSetMetaData.builder()
+                    .addColumn(new Column("CatalogName", ScalarType.createVarchar(30)))
+                    .addColumn(new Column("Type", ScalarType.createVarchar(30)))
+                    .addColumn(new Column("Props", ScalarType.createVarchar(8092)))
+                    .build();
```

Comment:
When specify a certain catalog, how about show result like:
|Key|Value|
|---|---|
|type | hms |
| key1 |value1|
|key2| value2|

That could be more easy to read?